client/src/components/new-request/analyze-pending.vue

```diff
@@ -83,4 +81,13 @@ export default class AnalyzePending extends Vue {
 
 </script>
 <style scoped lang="scss">
+@import '@/assets/scss/theme.scss';
+.detailed-search-title {
+  font-size: 1rem;
+  color: $gray9;
+  text-align: center;
+  padding: 0;
+  margin: 0;
+  padding-top: 1.5625rem;
+}
```

Comment:
We may have default styles (in base.scss) for some of these properties... but we can fix this later.